tests/end2end/playwright/edition-form.spec.js

```diff
@@ -112,3 +112,401 @@ test.describe('Edition Form Validation', () => {
       await expect(page.locator('#edition-form-container')).toBeHidden();
   })
 })
+
+
+test.describe('Multiple geometry layers', () => {
+
+    test.beforeEach(async ({ page }) => {
+        const url = '/index.php/view/map/?repository=testsrepository&project=multiple_geom';
+        await page.goto(url, { waitUntil: 'networkidle' });
```

Comment:
Please use the new helper `gotoMap(url, page)`

You should also set your IDE to clean trailing spaces. This will be soon required on the CI, see the commit d99a41fc9140b884853e8b5b749ac9a9c8c904c6 

Thanks ;-)